src/dev_deps.ts

```diff
@@ -8,4 +8,6 @@ export {
   fail,
 } from "https://deno.land/std@0.152.0/testing/asserts.ts";
 export * as mock from "https://deno.land/std@0.152.0/testing/mock.ts";
+export type { Spy } from "https://deno.land/std@0.177.0/testing/mock.ts";
```

Comment:
nit: Update the `mock` above to use the same latest lib version

Reply:
Now that I'm looking to update the std version, I noticed the following:

- On the STD README, under [Releases](https://deno.land/std@0.180.0#releases), it says "To check compatibility of different version of standard library with Deno CLI see [this list](https://raw.githubusercontent.com/denoland/dotland/main/versions.json)."
- [Said list](https://raw.githubusercontent.com/denoland/dotland/main/versions.json) (at the bottom) maps minimum deno version to different std versions

Sooooo, does this mean, when evaluating what version of the stdlib to use, we should take into consideration whatever minimum deno version we want to support for our users? gonna move this convo to a Slack thread...